js/main.js

```diff
@@ -13,236 +13,312 @@ const baseUrl =
     : "/";
 
 const BACKENDS = {
-  WebGPU: "WebGPU",
-  WebAssembly: "WebAssembly",
-  WebNN: "WebNN"
+  WASM: "WASM",
+  WEBGPU: "WebGPU",
+  WEBNN: "WebNN"
 };
 
-const SUPPORTED_DEVICES = {
-  GPU: "GPU",
+const DEVICES = {
   CPU: "CPU",
+  GPU: "GPU",
   NPU: "NPU"
 };
 
+// To add a new sample, please put <id>.png and <id>.mp4 in the public/assets folder.
```

Comment:
I hope all the fields are mandatory in the future, to make the maintenance easier.